### ISSUE TITLE
Javascript functions

### ISSUE THREAD

**Issue by gurnarok** [Luts91/Luts91.github.io#10]
Hi,
I was trying to read through the code and see, how it works. When I read the app.js, it doesn't make any sense, is it compressed or encoded?

**Comment by chunkybanana** [comment]
Yes, it's minified, but a lot of it seems to be raw data, with a lot of variables with strange names. I'm pretty sure it's also been uglified, and when [unminified](https://unminify.com/) it's 60k lines.
